components/select/style/index.tsx

```diff
@@ -144,6 +144,8 @@ const genBaseStyle: GenerateStyle<SelectToken> = (token) => {
 
       // ======================== Selection ========================
       [`${componentCls}-selection-item`]: {
+        display: 'flex',
+        alignItems: 'center',
```

Comment:
```suggestion
        alignItems: 'baseline',
```
Try `baseline` to align text.

Reply:
大佬我刚刚尝试了 `baseline`  ，但是文字省略依然失效